hack/jenkins/common.sh

```diff
@@ -289,6 +301,35 @@ elapsed=$min.$sec
 description="completed with ${status} in ${elapsed} minute(s)."
 echo $description
 
+echo ">> Copying ${TEST_OUT} to gs://minikube-builds/logs/${MINIKUBE_LOCATION}/${JOB_NAME}out.txt"
```

Comment:
`gs://minikube-builds/logs/${MINIKUBE_LOCATION}/${JOB_NAME}` seems to get re-used a lot in this script. Consider making it a variable.